src/client/extension.ts

```diff
@@ -202,7 +202,7 @@ export async function activate(
             if (artemisResponse !== null && artemisResponse.response !== null) {
                 ArtemisContext.setContext(artemisResponse);
 
-                const { geoName, geoLongName, clusterName, clusterNumber } = artemisResponse.response;
+                const { geoName, geoLongName, clusterName, clusterNumber, environment } = artemisResponse.response;
```

Comment:
should we create an object in common for this now - this is being used in multiple places - and list of params is increasing

Reply:
The response is already backed by a type `IArtemisAPIOrgResponse`. We are just destructuring it here.